chains/evm/contracts/interfaces/IFastTransferPool.sol

```diff
@@ -0,0 +1,75 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.0;
+
+/// @title IFastTransferPool
+/// @notice Interface for the CCIP Fast-Transfer Pool
+interface IFastTransferPool {
+  /// @notice Quote struct containing fee information
+  struct Quote {
+    uint256 sendTokenFee; // paid in feeToken
+    uint256 fastTransferFee; // paid in asset
```

Comment:
can these comments be more descriptive please. fastTransferFee makes it sound like that is the fee token not the other way around